optimum/intel/openvino/utils.py

```diff
@@ -137,8 +137,12 @@
     "automatic-speech-recognition-with-past": "OVModelForSpeechSeq2Seq",
 }
 
+CAUSAL_LANGUAGE_DATASETS = ["wikitext2", "c4", "c4-new", "auto"]
 
-LANGUAGE_DATASETS = ["wikitext2", "c4", "c4-new", "auto"]
+LANGUAGE_DATASETS = {
```

Comment:
could make sense to add the "PREDEFINED" prefix to match  the rest
```suggestion
PREDEFINED_LANGUAGE_DATASETS = {
```

Reply:
Thanks! Renamed